backends-common/cassandra/pom.xml

```diff
@@ -168,13 +168,25 @@
                     <version>${assertj-3.version}</version>
                     <scope>test</scope>
                 </dependency>
-		        <dependency>
-		            <groupId>org.cassandraunit</groupId>
-		            <artifactId>cassandra-unit</artifactId>
-		            <version>2.1.9.2</version>
-		            <scope>test</scope>
-		        </dependency>
-		    </dependencies>
+                <dependency>
+                    <groupId>org.cassandraunit</groupId>
+                    <artifactId>cassandra-unit</artifactId>
+                    <version>2.1.9.2</version>
+                    <scope>test</scope>
+                </dependency>
+                <dependency>
+                    <groupId>org.mockito</groupId>
+                    <artifactId>mockito-core</artifactId>
+                    <version>1.10.19</version>
+                    <scope>test</scope>
+                </dependency>
+                <dependency>
+                    <groupId>uk.org.lidalia</groupId>
+                    <artifactId>slf4j-test</artifactId>
```

Comment:
I think here logging IS IMPORTANT. It is important to be sure we log about too old schema version no?
When you slf4j-test, you have to make sure it is the only implementation of slf4j in the scope so you can't have logback.